lighthouse-extension/app/src/popup.js

```diff
@@ -128,6 +128,10 @@ document.addEventListener('DOMContentLoaded', _ => {
     return frag;
   }
 
+  if (background.isRunning()) {
+    startSpinner();
```

Comment:
not super familiar with the code here, but i'm assuming we're already stopping the spinner everywhere we need to? :)